### PR TITLE
Add "Resource Recommendations"

### PR DESCRIPTION
**Pull request by davidhammaker**
This PR adds the "Resource Recommendations" article, plus a few very minor tweaks to two of the other articles.

### CODE REVIEW

File: _posts/2024-01-02-resource-recommendations.html [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+<p>
+  These resources have also formed and shaped Upstream. We’ve allowed these
+  books to influence how we think and approach ministry. In this sense, the list
+  also gives a bit of a behind the scenes look into who we are and what we
+  value.
+</p>
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
+    <img src="{{ "/assets/images/articles/resource_the-five-disfunctions.jpeg" |
```
Comment (ndirig) [inline _posts/2024-01-02-resource-recommendations.html:32]: The image width should be the same as the article body text.

From design:
Screenshot 2024-04-17 at 9 39 35 AM

vs:
Screenshot 2024-04-17 at 9 40 18 AM

File: _posts/2024-01-02-resource-recommendations.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
+---
+author:
+duration: 4 Min Read
```
Comment (ndirig) [inline _posts/2024-01-02-resource-recommendations.html:3]: Should be "Upstream"
Screenshot 2024-04-17 at 9 42 11 AM

File: _posts/2024-01-02-resource-recommendations.html (hunk 1 above)
Comment (ndirig) [reply _posts/2024-01-02-resource-recommendations.html:32]: I now realize these comments are under the wrong image, in the wrong file (this adjustment will need to be in _layouts/article.html, line 17).

This change would affect all other articles, so hopefully this is an easy fix across the board!

File: _posts/2024-01-02-resource-recommendations.html [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+</p>
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
+    <img src="{{ "/assets/images/articles/resource_the-five-disfunctions.jpeg" |
+    relative_url }}" class="img-fluid resource-img" alt="The Five Dysfunctions of
+    a Team: A Leadership Fable">
+  </div>
+  <div class="content-box order-2 col-12 col-md-7 resource-detail">
+    <p class="resource-topic">Team DNA and Community</p>
```
Comment (ndirig) [inline _posts/2024-01-02-resource-recommendations.html:38]: This should use overline font styling.
Screenshot 2024-04-17 at 9 47 06 AM

vs
Screenshot 2024-04-17 at 9 48 12 AM

File: _posts/2024-01-02-resource-recommendations.html [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+<p>
+  These resources have also formed and shaped Upstream. We’ve allowed these
+  books to influence how we think and approach ministry. In this sense, the list
+  also gives a bit of a behind the scenes look into who we are and what we
+  value.
+</p>
+
+<div class="row justify-content-between align-items-center">
+  <div class="start-image-box order-1 col-12 col-md-5 text-center">
```
Comment (ndirig) [inline _posts/2024-01-02-resource-recommendations.html:31]: The padding between the photo and the text could be a bit more.

From design:
Screenshot 2024-04-17 at 9 50 26 AM

vs:
Screenshot 2024-04-17 at 9 49 19 AM

File: _posts/2024-01-02-resource-recommendations.html (hunk 3 above)
Comment (ndirig) [reply _posts/2024-01-02-resource-recommendations.html:38]: Same comments apply to other resources

**Review by ndirig — CHANGES_REQUESTED** [review]
Just a few minor requests. This looks great otherwise!

**Comment by davidhammaker** [comment]
> Just a few minor requests. This looks great otherwise!

I've added the author, updated the font, and fixed the padding! I didn't touch the article layout yet -- I figure that might be part of a different PR that more comprehensively covers style fixes across the site. Let me know if this is good to go!

**Review by ndirig — APPROVED** [review]

**Pull request merged**